.github/workflows/ci.yml

```
@@ -143,6 +181,53 @@ jobs:
+        run: pip install pip-audit
+      - name: Run pip-audit
+        run: |
+          # Upgrade mistune to fix CVEs
+          pip install --upgrade mistune>=3.2.1
+          # Ignore environment-specific pip vulnerabilities (upstream issues)
+          pip-audit --ignore-vuln CVE-2025-8869 --ignore-vuln CVE-2026-1703 --ignore-vuln CVE-2026-3219 --ignore-vuln CVE-2026-6357
+      - name: Upload Test Results
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: test-results-${{ matrix.os }}-${{ matrix.python-version }}
```

actionlint (1.7.12)

[error] property "os" is not defined in object type {}

(expression)